pkg/controller/ibmblockcsi/syncer/csi_node.go

```diff
@@ -299,6 +309,8 @@ func (s *csiNodeSyncer) ensureVolumes() []corev1.Volume {
 		ensureVolume("device-dir", ensureHostPathVolumeSource("/dev", "Directory")),
 		ensureVolume("sys-dir", ensureHostPathVolumeSource("/sys", "Directory")),
 		ensureVolume("host-dir", ensureHostPathVolumeSource("/", "Directory")),
+		ensureVolume("lib-modules", ensureHostPathVolumeSource("/lib/modules", "Directory")),
+		ensureVolume("iscsi", ensureHostPathVolumeSource("/etc/iscsi", "DirectoryOrCreate")),
```

Comment:
remove `OrCreate` (this was [never reviewed](https://github.com/IBM/ibm-block-csi-operator/pull/1))